crates/database/src/orders.rs

```diff
@@ -404,6 +409,7 @@ pub struct FullOrder {
     pub buy_token_balance: BuyTokenDestination,
     pub presignature_pending: bool,
     pub pre_interactions: Vec<(Address, BigDecimal, Vec<u8>)>,
+    pub post_interactions: Vec<(Address, BigDecimal, Vec<u8>)>,
     pub ethflow_data: Option<(Option<TransactionHash>, i64)>,
```

Comment:
I know this isn't on you, but it would be really nice to get rid of these tuples and either name these fields or give strong types to the tuple elements. It's hard to tell at a glance what the `BigDecimal` and `i64` are for. And since this is a public signature being able to deduce as much as possible at a glance is important.

Reply:
Unless I misunderstood the comment on the big SQL query this unfortunate type is due to limitations in `sqlx`. I created a type alias and attached the related comments to it for now.

Reply:
I _think_ you should be able to use a struct with `derive(sqlx::FromRow)` right? (May be wrong though).

Reply:
This currently doesn't work and there is an open [issue](https://github.com/launchbadge/sqlx/issues/298) about it.